src/common/legacy.ts

```diff
@@ -122,7 +122,8 @@ export const getRaceData = async (
         handicapWeight: item.handicapWeight,
         proposition_id: proposition_id,
         signature: signature,
-        barrier: item.barrierNumber
+        barrier: item.barrierNumber,
+        bettingStatus: item.fixedOdds.bettingStatus
```

Comment:
I agree with @brettvniekerk 

```suggestion
        status: item.fixedOdds.bettingStatus
```